fe/fe-core/src/main/java/org/apache/doris/datasource/paimon/PaimonSchemaCacheValue.java

```diff
@@ -20,18 +20,42 @@
 import org.apache.doris.catalog.Column;
 import org.apache.doris.datasource.SchemaCacheValue;
 
+import org.apache.paimon.schema.TableSchema;
+import org.apache.paimon.types.DataField;
+
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class PaimonSchemaCacheValue extends SchemaCacheValue {
 
     private List<Column> partitionColumns;
 
-    public PaimonSchemaCacheValue(List<Column> schema, List<Column> partitionColumns) {
+    private TableSchema tableSchema;
```

Comment:
Looks like we only use this `TableSchema` to build `columnIdToName`, no need to store it.